dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -17,15 +17,30 @@
 #include "utils/datetime.h"
 #include "utils/fileutils.h"
 #include "utils/signal-monitor.h"
+#include "utils/stringutils.h"
+
+static struct rumcontext {
+    char application_id[200];
+    char session_id[200];
+    char view_id[200];
+} rumcontext;
+
+static struct maincontext {
+    const char *generic_message;
+    const char *emergency_status;
+    const char *error_kind;
+    const char *logger_name;
+    char environment[200];
+    char service_name[200];
+    char storage_dir[200];
+} context{"Native crash detected",
```

Comment:
nit: add a space between `context` and `{` and put the string on a new line.

Reply:
You should also probably add a comment about why you have 7 fields defined but you only provide initialization for 5